### PR TITLE
Added post semantics in html

### PR DESCRIPTION
**Pull request by maida12**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -445,6 +445,19 @@ <h1>HTML MathML</h1>
...
+            <div class="main__html-componentBox">
+                <div class="main__html-componentBox-logobox">
+                    <img src="/Website/assets/svg/icons8-html-5.svg" alt="html logo">
+                </div>
+
+                <div class="main__html-componentBox-textBox">
+                    <h1>Semantics In HTML</h1>
+                    <p>
+                       Semantics gives meaning to a document and helps search engines to better understand what information is contained in web pages.
+                        <a href="/html/html_math/">Learn More <i class="bi bi-book"></i></a>
```
Comment (Diptenusarkar) [inline index.html:457]:
``` suggestion
                        <a href="/html/html_semantics/">Learn More <i class="bi bi-book"></i></a>
```

File: html/html_semantics/index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                  <li>Use tables for tabular data: Tables (<code>&lt;table&gt;</code>, <code>&lt;thead&gt;</code>, <code>&lt;tbody&gt;</code>, <code>&lt;tfoot&gt;</code>, <code>&lt;tr&gt;</code>, <code>&lt;th&gt;</code>, and <code>&lt;td&gt;</code>) are used to display tabular data. It is important to use the correct table structure to ensure that the data is presented in a clear and accessible way.</li>
+                </ul>
+            
+                I hope this helps you understand the basics of HTML semantics.
+              </p>
+            <hr>
+        </div>
+
+        <div class="navigation-link">
+            <a href="#"><i class="bi bi-chevron-double-left"></i> Previous</a>
```
Comment (Diptenusarkar) [inline html/html_semantics/index.html:118]:
```suggestion
            <a href="../html_math/"><i class="bi bi-chevron-double-left"></i> Previous</a>
```

File: html/html_semantics/index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            <a href="#">Next <i class="bi bi-chevron-double-right"></i></a>
```
Comment (Diptenusarkar) [inline html/html_semantics/index.html:119]:
```suggestion
            <a href="/">Next <i class="bi bi-chevron-double-right"></i></a>
```

File: html/html_semantics/index.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        <select id="themeSwitcher">
+            <option value="">Select One</option>
+            <option value="theme-dark">Dark</option>
+            <option value="theme-light">Light</option>
+        </select>
+    </div> -->
+
+    <main class="main">
+        <div class="navigation-link">
+            <a href="#"><i class="bi bi-chevron-double-left"></i> Previous</a>
```
Comment (Diptenusarkar) [inline html/html_semantics/index.html:90]:
``` suggestion
            <a href="../html_math/"><i class="bi bi-chevron-double-left"></i> Previous</a>
```

File: html/html_semantics/index.html [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+            <a href="#">Next <i class="bi bi-chevron-double-right"></i></a>
```
Comment (Diptenusarkar) [inline html/html_semantics/index.html:91]:
```suggestion
            <a href="/">Next <i class="bi bi-chevron-double-right"></i></a>
```

**Review by Diptenusarkar — CHANGES_REQUESTED** [review]
Need to comply with this code example.

**Comment by maida12** [comment]
I did all these changes

**Comment by Diptenusarkar** [comment]
something is not working

**Comment by maida12** [comment]
should i make a new pull request?

**Pull request closed without merging**